backend/internal/usecase/auth_usecase.go

```
@@ -0,0 +1,500 @@
+	users repository.AuthUserRepository,
+	tokens repository.PasswordResetTokenRepository,
+) *RequestPasswordResetUseCase {
+	return &RequestPasswordResetUseCase{users: users, tokens: tokens}
+}
+
+// Execute はリセットトークン（平文）を返す。
+// メールアドレスの存在有無を漏らさないため、未登録でもエラーにせず空文字を返す。
+func (uc *RequestPasswordResetUseCase) Execute(ctx context.Context, email string) (string, error) {
+	user, err := uc.users.FindByEmail(ctx, email)
+	if err != nil || user == nil {
+		return "", nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 437) but it returns nil

(nilerr)